code/modules/mob/living/carbon/xenomorph/XenoProcs.dm

```diff
@@ -26,7 +26,7 @@
 
 		var/area/A = get_area(X)
 		xenoinfo += " <b><font color=green>([A ? A.name : null])</b></td></tr>"
-	
+
```

Comment:
this file has been renamed

Reply:
It appears as lower case xenoprocs on my end; weird.